strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/layout/pypi/PypiBrowsePackageHtmlResponseBuilder.java

```diff
@@ -0,0 +1,69 @@
+package org.carlspring.strongbox.controllers.layout.pypi;
+
+import org.carlspring.strongbox.artifact.coordinates.PypiArtifactCoordinates;
+import org.carlspring.strongbox.providers.io.RepositoryFiles;
+import org.carlspring.strongbox.providers.io.RepositoryPath;
+import org.carlspring.strongbox.storage.repository.Repository;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+/**
+ * Helper class for {@link PypiArtifactController}
+ * 
+ * @author ankit.tomar
+ */
+@Component
+public class PypiBrowsePackageHtmlResponseBuilder
+{
+
+    private final Logger logger = LoggerFactory.getLogger(PypiBrowsePackageHtmlResponseBuilder.class);
+
+    public String getHtmlResponse(List<Path> filePaths,
+                                  String packageName,
```

Comment:
we can probably get the `packageName` from `PypiArtifactCoordinates` as well. And also it will be good to check that all paths have the same `packageName`.

Reply:
package can have `-` in name whereas in `whl` artifact it is replaced with `_` and in html response original package name is expected .
Eg: `hello-world` passed in request is converted to `hello_world` for querying database.

Reply:
I suggest to have a utility method that convert package name forward and backward.

Reply:
Done.